### ISSUE TITLE
More control of hyperparams

### ISSUE THREAD

**Issue by antoninodimaggio** [antoninodimaggio/Hugging-Captions#9]
- Learning rate, number of epochs, and so on
- This may make the app less usable for the casual user.
